HomeAssistant.xcodeproj/project.pbxproj

```diff
@@ -6923,6 +6928,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 675CE4281FE5F1920B13D553 /* Pods-iOS-App.debug.xcconfig */;
 			buildSettings = {
+				ASSETCATALOG_COMPILER_ALTERNATE_APPICON_NAMES = "trans beta dev black caribbean-green blue cornflower-blue electric-violet crimson green classic old-beta old-dev old-release fire-orange orange purple pink white red bi_pride release POC_pride non-binary rainbow";
```

Comment:
@zacwest I couldn't find a way to specify the asset catalog itself, if you have done that before let me know the best way

Reply:
i can't seem to find it either. i think you are right that `ASSETCATALOG_COMPILER_INCLUDE_ALL_APPICON_ASSETS` is a better approach. how about - let's push up a testflight build and see if the app store app sizing says it's different or not. maybe the AS will strip the 1024x1048 file?

Reply:
Sounds good, I reverted my change, let's see how it goes